Bot upgrades golang packages to Invalid pseudo-version

**What Renovate type are you using?**
Renovate GitHub App

**Describe the bug**
Renovate is picking the wrong timestamp when it upgrades a golang package. It picks the commit timestamp of the main go package not the commit timestamp of the dependency.
So here https://github.com/mgechev/revive/pull/221/. 

the bot updated at 2019-08-15 the package `golang.org/x/sys` like this
```
golang.org/x/sys v0.0.0-20190815002308-fde4db37ae7a
```
But actually it should pick the date from here https://github.com/golang/sys/commit/fde4db37ae7ad8191b03d30d27f258b5291ae4e3 which is 2019-08-13. Even if you do the upgrade manually on your laptop it will be 
```
go get -u golang.org/x/sys
go: finding golang.org/x/sys latest
go: downloading golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
go: extracting golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a
```

**To Reproduce**

1.  Create a simple go package with `main.go`
```
package main

import (
    "golang.org/x/sys/unix"
)

func main() {
    print(unix.Getpid())
}
```
2.  And do the following to build go.mod and go.sum
```
$ export GO111MODULE=on
$ go mod init
$ go run main.go
```

3. Check go.mod and go.sum

```
// go.mod

go 1.12

require golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a // indirect
```
```
// go.sum
golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a h1:aYOabOQFp6Vj6W1F80affTUvO9UxmJRx8K0gsfABByQ=
golang.org/x/sys v0.0.0-20190813064441-fde4db37ae7a/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
```
4. The bot will actually upgrade to the latest commit but with wrong timestamp (mostly the current timestamp) like here https://github.com/mgechev/revive/pull/221/files
```
golang.org/x/sys v0.0.0-20190815002308-fde4db37ae7a // indirect
```
```
golang.org/x/sys v0.0.0-20190815002308-fde4db37ae7a h1:ZtbkXZY7vbjKdcaQck5BbF6wn7cKkfzW9xhtHmXev8A=
golang.org/x/sys v0.0.0-20190815002308-fde4db37ae7a/go.mod h1:h1NjWce9XRLGQEsW7wpKNCjG9DtNlClVuFLEZdDNbEs=
```

I think this line cause the issue
https://github.com/renovatebot/renovate/blob/6f99118f7cfb9a646148455938793c8894d10a01/lib/manager/gomod/update.ts#L60-L64

**Additional context**
This is fine now but once golang 1.13 released which is soon, there will be more troubles since go will validate the timestamp during package installation (See here https://tip.golang.org/doc/go1.13#version-validation) and when i try to use packages like revive that depend on the bot for upgrades with latest go version 1.13, things fail with error 
```
go get github.com/mgechev/revive
go: finding github.com/mgechev/revive latest
go: downloading github.com/mgechev/revive v0.0.0-20190817163854-e8a54f5ffbbb
go: extracting github.com/mgechev/revive v0.0.0-20190817163854-e8a54f5ffbbb
go get: github.com/mgechev/revive@v0.0.0-20190817163854-e8a54f5ffbbb requires
	golang.org/x/sys@v0.0.0-20190815002308-fde4db37ae7a: invalid pseudo-version: does not match version-control timestamp (2019-08-13T06:44:41Z)
Makefile:16: recipe for target 'install_revive' failed
```

--- a/lib/manager/gomod/update.ts
+++ b/lib/manager/gomod/update.ts
@@ -1,4 +1,3 @@
-import { DateTime } from 'luxon';
 import { logger } from '../../logger';
 import { Upgrade } from '../common';
 
@@ -57,9 +56,10 @@ export function updateDependency(
         { depName, lineToChange, newDigestRightSized },
         'gomod: need to update digest'
       );
-      const currentDateTime = DateTime.local().toFormat('yyyyMMddHHmmss');
-      const newValue = `v0.0.0-${currentDateTime}-${newDigestRightSized}`;
-      newLine = lineToChange.replace(updateLineExp, `$1$2${newValue}`);
+      newLine = lineToChange.replace(
+        updateLineExp,
+        `$1$2${newDigestRightSized}`
+      );
     } else {
       newLine = lineToChange.replace(updateLineExp, `$1$2${upgrade.newValue}`);
     }

